tutor/src/screens/task/step/fr-nudge-help.js

```diff
@@ -37,8 +37,8 @@ const Actions = styled.div`
 const FrNudgeHelp = observer(({ course, step, ux }) => {
   return (
     <StyledFrNudgeHelp>
-      <Title>Need Help?</Title>
-      <Msg>{ux.nudgeMessage}</Msg>
+      <Title>{ux.nudge.title}</Title>
```

Comment:
Hey Nathan, can this cause a type error since ux.nudge can be null?

Reply:
Good thought, but it shouldn't ever be null.  the ux class assigns the nudge in it's constructor by picking a random one from the choices

Reply:
ahhhhhhh okay, thanks for explaining that!